go/controller/internal/controller/common.go

```diff
@@ -48,6 +51,10 @@ var (
 	waitForResources = ctrl.Result{RequeueAfter: requeueWaitForResources}
```

Comment:
Should we also add jitter when it's waiting for resources? It's used in a lot of cases as `handleRequeue` uses it.

Reply:
yea we definitely should i just didn't get to it.  I'd add another ticket to handle it